init.bash

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+# Find the absolute script location dir (e.g. BASEDIR=/opt/openhabian)
+SOURCE="${BASH_SOURCE[0]}"
+while [ -h "$SOURCE" ]; do
+  BASEDIR="$( cd -P "$( dirname "$SOURCE" )" && pwd )"
+  SOURCE="$(readlink "$SOURCE")"
+  [[ $SOURCE != /* ]] && SOURCE="$BASEDIR/$SOURCE"
+done
+BASEDIR="$( cd -P "$( dirname "$SOURCE" )" && pwd )"
+SCRIPTNAME="$(basename "$SOURCE")"
+
+CONFIGFILE="/etc/openhabian.conf"
+REPOSITORYURL=$(grep -i '^repositoryurl' ${CONFIGFILE} | cut -d '=' -f2)
+CLONEBRANCH=$(grep -i '^clonebranch' ${CONFIGFILE} | cut -d '=' -f2)
+JAVA_ARCH=$(grep -i '^java_arch' ${CONFIGFILE} | cut -d '=' -f2)
+MODE=$(grep -i '^mode' /etc/openhabian.conf | cut -d '=' -f2)
```

Comment:
I used uppercase to ensure we only ever use what we obtain through init.bash.
Frankly I'm not sure we "source the config" as I have seen unset (lowercase) variables at times,
that's why this whole init.bash introduction

Reply:
actually I do not like having both versions (via init.bash in capital letters and via sourcing the config) - that is what I have seen in first-boot.bash.
I would actually prefer not to source any config file. sourcing could be removed from first-boot.bash (if something is missing, lets add it to init.bash).

Reply:
Granted it's doing things twice. I'll see to remove init.bash and go back to lowercase variables sourced from openhabian.conf.
Will also reorder sourcing in first-boot.bash (I saw you were working on it)

Reply:
I rather decided myself to leave sourcing in because it was always built to work like that. 
Besides sourcing is a clever sort of parsing a config file, isn't it ? If there's anything that we would need to store in .conf that's incompatible with bash syntax , we can still change it and move over to init.bash, but for the time being I'll keep it. Don't want to upset Thomas :)

Reply:
Yes, parsing the content could be beneficial, error messages are always better than error hiding. Though, we never should give write permissions to openhab user (maybe also warn about deviations on startup). Otherwise one could inject code. Currently we are safe (755).